docs/content/stepbystep/IDE-setup.mdx

```diff
@@ -0,0 +1,45 @@
+---
+title: 'Setting up the IDE'
+metaTitle: 'IDE Setup | Discordeno'
+metaDescription: 'Setup VSC for discordeno'
+---
+
+# Installing
+
+There are many editors for js/ts, but I recommend Visual Studio Code as its free and fast with a lot of extensions (Including a Deno one)
+
+Here are the others:
+- WebStorm
+- Vim
+- Emacs
+- Atom
+
+First, go to <https://code.visualstudio.com> and download the installer, the installation is really easy. Check out the gif below
+
+![installing gif](https://i.imgur.com/xvDJ2iI.gif)
+
+Once that is done open up VSC(This is how we will refer to it from now on). It might take a while as it is loading the app for the first time and has not cached things yet.
+
+# Making it look better
+
+Lets install a theme and an icon pack to make VSC look a little better. None of this is required
+
+First, go to the extensions pane
+
+![extensions button](https://i.imgur.com/Hw0pY2z.png)
+
+Then search for material and install the theme and icons. I like the Material Ocean Theme
+
+![installing the material theme](https://i.imgur.com/DrnvBJS.png)
+
+# Installing the Deno extension
+
+VSC does not have Deno support by default, so we will add the official extension
+
+Go to extensions then search for deno, then click install
+
+![searching and installing Deno](https://i.imgur.com/G0CwSiS.gif)
+
+(You might also want the std snippets if your thinking of using deno for other things)
```

Comment:
Missing period at the end of the sentence.

Reply:
Put like **Advanced:** so they know its not meabnt for beginners

Reply:
Hmm... good suggestion!